### DISCUSSION TITLE
A walkthrough of examples using reverse ETL tools with dbt for improved marketing flows

### DISCUSSION THREAD

**Discussion by gwenwindflower** [dbt-labs/docs.getdbt.com#1149] [Archive]
@krevitt has already done substantial work on this one -- transferred from the [archived developer blog repo](https://github.com/dbt-labs/dbt-technical-blog-writing/issues/17)

**Comment by krevitt** [comment]
[Draft getting going here] - needs an outline + figjam structure as an exemplar (https://docs.google.com/document/d/16da097qjEn3hCNzcDnwzw1akc0S5dZqZUuTbNV303xQ/edit#heading=h.ienofcuhofu6)
